### ISSUE TITLE
StoryArcs: Check for watchlist matches failing

### ISSUE THREAD

**Issue by kurnalbert** [evilhero/mylar#1250]
When attempting to check for watchlist matches on a story arc, it is instantly popping up a red box saying error in the bottom right hand corner of Mylar. It does not pick up any of the issues which are already in my watchlist that were downloaded and post-processed by Mylar.

Below is the log it produces as soon as the button is clicked:

14-Apr-2016 17:07:55 - DEBUG :: CP Server Thread-8 : arc: Standoff : Avengers Standoff: Welcome to Pleasant Hill : 1
14-Apr-2016 17:07:55 - DEBUG :: CP Server Thread-8 : initial name match - confirming issue # is present in series
14-Apr-2016 17:07:55 - DEBUG :: CP Server Thread-8 : int_issue = 1000
14-Apr-2016 17:07:55 - DEBUG :: CP Server Thread-8 : COMIC-comicyear: 2016
14-Apr-2016 17:07:55 - DEBUG :: CP Server Thread-8 : ARC-seriesyear: 2016
14-Apr-2016 17:07:55 - DEBUG :: CP Server Thread-8 : issue #: 1 is present!
14-Apr-2016 17:07:55 - DEBUG :: CP Server Thread-8 : isschk: <sqlite3.Row object at 0x10312faf0>
14-Apr-2016 17:07:55 - DEBUG :: CP Server Thread-8 : Comicname: Avengers Standoff: Welcome to Pleasant Hill
14-Apr-2016 17:07:55 - DEBUG :: CP Server Thread-8 : ComicID: 88187
14-Apr-2016 17:07:55 - DEBUG :: CP Server Thread-8 : Issue: 1
14-Apr-2016 17:07:55 - DEBUG :: CP Server Thread-8 : IssueArcID: 58741_1539
14-Apr-2016 17:07:55 - DEBUG :: CP Server Thread-8 : arc: Standoff : Avengers Standoff: Assault On Pleasant Hill Alpha : 1
14-Apr-2016 17:07:55 - DEBUG :: CP Server Thread-8 : Unable to find a match for Avengers Standoff: Assault On Pleasant Hill Alpha :#1
14-Apr-2016 17:07:55 - DEBUG :: CP Server Thread-8 : destination location set to  : /Volumes/OldMac/eBooks/Comics/StoryArcs/Standoff
14-Apr-2016 17:07:55 - INFO :: CP Server Thread-8 : there are 0 files.
14-Apr-2016 17:07:55 - DEBUG :: CP Server Thread-8 : files in directory: 0

**Comment by evilhero** [comment]
Yep, story-arcs are abit borked atm - the huge change in the file checker module impacts this, so I'm looking at this now, just not sure about a timeframe for fix atm (shouldn't be longer than end of day I'd imagine, but atm I'm still working on things).

**Comment by kurnalbert** [comment]
Just tried again after the last update and the issue still persists. Thank you for all of your work on this!

Logs as follows:
15-Apr-2016 20:43:53 - INFO :: CP Server Thread-6 : there are 1 files.
15-Apr-2016 20:43:53 - INFO :: CP Server Thread-6 : [FILENAME]: 010-Illuminati 006 (2016).cbz
15-Apr-2016 20:43:53 - DEBUG :: CP Server Thread-6 : checking date : 2016
15-Apr-2016 20:43:53 - DEBUG :: CP Server Thread-6 : date: 2016
15-Apr-2016 20:43:53 - DEBUG :: CP Server Thread-6 : datecheck: [{'date': '2016', 'position': 2, 'mod_position': 15}]
15-Apr-2016 20:43:53 - DEBUG :: CP Server Thread-6 : sctd: 2016
15-Apr-2016 20:43:53 - DEBUG :: CP Server Thread-6 : year verified as : 2016
15-Apr-2016 20:43:53 - DEBUG :: CP Server Thread-6 : year verified as: 2016
15-Apr-2016 20:43:53 - DEBUG :: CP Server Thread-6 : highest_series_position: 2
15-Apr-2016 20:43:53 - DEBUG :: CP Server Thread-6 : possible_issuenumbers: [{'position': 1, 'validcountchk': False, 'number': '006', 'mod_position': 11}]
15-Apr-2016 20:43:53 - DEBUG :: CP Server Thread-6 : issue verified as : 006
15-Apr-2016 20:43:53 - DEBUG :: CP Server Thread-6 : series title possibly: Illuminati
15-Apr-2016 20:43:53 - DEBUG :: CP Server Thread-6 : [STORY ARC DIRECTORY] 1 files exist within this directory.
15-Apr-2016 20:43:53 - INFO :: CP Server Thread-6 : {'comiclist': [{'issue_number': '006', 'sub': None, 'issue_year': '2016', 'comiclocation': u'/Volumes/OldMac/eBooks/Comics/StoryArcs/Standoff', 'scangroup': None, 'dynamic_name': 'Illuminati', 'series_name': 'Illuminati', 'comicfilename': '010-Illuminati 006 (2016).cbz', 'series_volume': None}], 'comiccount': 1}
15-Apr-2016 20:43:53 - DEBUG :: CP Server Thread-6 : [Standoff] Avengers Standoff: Welcome to Pleasant Hill : 1
